### PR TITLE
fix(curriculum): external straddling link in cip

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

clear task 98 in#40749

image

### CODE REVIEW

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ dashedName: straddling-checkerboard
 
 # --description--
 
-Implement functions to encrypt and decrypt a message using the [straddling checkerboard](https://en.wikipedia.org/wiki/Straddling_checkerboard) method. The functions will take a string and an array as parameters. The array has 3 strings representing the 3 rows of the checkerboard. The output will be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit unencrypted. This should be reversed for decryption.
+Implement functions to encrypt and decrypt a message using the **straddling checkerboard** method.
+
+- The **functions** will take a <em>string</em> and an <em>array</em> as parameters.
```
Comment (Sembauke) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md:13]: should the word functions be plural in this case?

Comment (Sboonny) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md:13]: I thought functions refer to the seed content

```js
function straddle(message, alphabet) {

}

function unstraddle(message, alphabet) {

}
```

If I am misunderstanding, I am happy to fix it.

**Review by Sembauke — DISMISSED** [review]

**Comment by scissorsneedfoodtoo** [comment]
Hi @Sboonny, thanks for all of your help removing these external links, and for improving the challenge descriptions.

I have a couple of minor suggestions I'll leave as review comments.

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md [hunk 2]
```diff
@@ -8,7 +8,11 @@ dashedName: straddling-checkerboard
 
 # --description--
 
-Implement functions to encrypt and decrypt a message using the [straddling checkerboard](https://en.wikipedia.org/wiki/Straddling_checkerboard) method. The functions will take a string and an array as parameters. The array has 3 strings representing the 3 rows of the checkerboard. The output will be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit unencrypted. This should be reversed for decryption.
+Implement functions to encrypt and decrypt a message using the **straddling checkerboard** method.
+
+Each function will take a two arguments, a `message` string and  the `alphabet` array. The `alphabet` contains 3 strings representing the 3 rows of the checkerboard.
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md:13]:
```suggestion
Each function will take two arguments, a `message` string and an `alphabet` array. The `alphabet` array will contain 3 strings representing the 3 rows of the straddling checkerboard.
```

File: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md [hunk 3]
```diff
@@ -8,7 +8,11 @@ dashedName: straddling-checkerboard
 
 # --description--
 
-Implement functions to encrypt and decrypt a message using the [straddling checkerboard](https://en.wikipedia.org/wiki/Straddling_checkerboard) method. The functions will take a string and an array as parameters. The array has 3 strings representing the 3 rows of the checkerboard. The output will be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit unencrypted. This should be reversed for decryption.
+Implement functions to encrypt and decrypt a message using the **straddling checkerboard** method.
+
+Each function will take a two arguments, a `message` string and  the `alphabet` array. The `alphabet` contains 3 strings representing the 3 rows of the checkerboard. 
+
+The output of the `straddle()` function is a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit <em>unencrypted</em>. This should be reversed for decryption with the `unstraddle()` function.
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md:15]:
```suggestion
The output of the `straddle()` function should be a series of decimal digits. Numbers should be encrypted by inserting the escape character before each digit, then including the digit <em>unencrypted</em>.

This should be reversed for decryption with the `unstraddle()` function.
```

Comment (Sboonny) [reply curriculum/challenges/english/10-coding-interview-prep/rosetta-code/straddling-checkerboard.md:15]: I do like the suggestions, I am implementing them

**Review by scissorsneedfoodtoo — APPROVED** [review]

**Review by Sembauke — APPROVED** [review]

**Pull request merged**